Spanner/src/Instance.php

```diff
@@ -216,7 +216,15 @@ public function info(array $options = [])
     public function exists(array $options = [])
     {
         try {
-            $this->reload($options = []);
+            if ($this->info) {
+                $this->connection->getInstance($options + [
```

Comment:
I'm sorry for not raising this in the previous commit. Please put `$options` last so that the required arguments take precedence.

```php
				$this->connection->getInstance([
                    'name' => $this->name,
                    'projectId' => $this->projectId,
                    'fieldMask' => ['name'],
                ] + $options);
```

We don't want users to be able to override `name`, `projectId` or `fieldMask`.

Using the `+` operator on arrays uses left-hand precedence:

```sh
php > var_dump(['foo' => 'a'] + ['foo' => 'b']);
php shell code:1:
array(1) {
  'foo' =>
  string(1) "a"
}
```

This isn't universally applied across the library; some old instances are incorrect, but going forward please try to apply this.